10/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -502,3 +510,12 @@ process_extending_files()
     done
   done <<<"$(get_matched_files '*.sh' "$@" | sort -u)"
 }
+
+create_extensions()
+{
+  if [ -v POSTGRESQL_EXTENSIONS ]; then
+    for EXT in $POSTGRESQL_EXTENSIONS; do
+      psql -c "CREATE EXTENSION IF NOT EXISTS ${EXT};"
+    done;
```

Comment:
```suggestion
    done
```
This is typo, I guess.